pybleno/Bleno.py

```diff
@@ -16,6 +15,7 @@
 else:
     raise Exception('Unsupported platform')
 
+from .hci_socket.Emit import Emit
```

Comment:
Import moved after L13 because the glob import results in `Emit` referring to the module, not the class.